pkg/network/ovn_kubernetes.go

```diff
@@ -349,6 +349,12 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 		data.Data["NorthdThreads"] = 4
 	}
 
+	if conf.DisableMultiNetwork != nil && !*conf.DisableMultiNetwork {
+		data.Data["OVN_MULTI_NETWORK_ENABLE"] = true
+	} else {
+		data.Data["OVN_MULTI_NETWORK_ENABLE"] = false
+	}
+
```

Comment:
nit: double negation

```
	if conf.DisableMultiNetwork != nil && *conf.DisableMultiNetwork {
		data.Data["OVN_MULTI_NETWORK_ENABLE"] = false
	} else {
		data.Data["OVN_MULTI_NETWORK_ENABLE"] = true
	}
```

Reply:
fair enough.

Would it make more sense to have @jcaamano ? 
```golang
data.Data["OVN_MULTI_NETWORK_ENABLE"] = true
if conf.DisableMultiNetwork != nil && *conf.DisableMultiNetwork {
    data.Data["OVN_MULTI_NETWORK_ENABLE"] = false
}
```

Reply:
Done.